lesson07/assignment/spaceland/src/components/ContactPage.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+<div class="contact">
+  <h2 class="text-danger text-center m-4 p-5">{{ msg }}</h2>
+
+  <body>
+    <main class="main">
+    <h3 class="text-primary text-center m-4 p-6">Please feel free to reach out through the following:</h3>
+    <ul class="text-center list-inline">
+      <li class="list-inline-item p-3" v-for="item in items" v-bind:key="item">
+        {{item.text}}
+        <div v-if="item.checked" class="text-warning">Available</div>
+        <div v-else>Not Availavble</div>
+      </li>
+    </ul>
+      <div class="m-5" id="accordion">
+        <div class="card m-5">
+          <div class="card-header" id="headingOne">
+            <h5 class="mb-0">
+              <button class="btn btn-link" data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+                Contact Us
+              </button>
+            </h5>
+          </div>
+
+          <div id="collapseOne" class="collapse" aria-labelledby="headingOne" data-parent="#accordion">
```

Comment:
We're looking for you to loop through an array of objects for your accordion. So you should have more than one accordion. I would also suggest using bootstrap's b-collapse for your accordion, it'd make things a bit easier for you. The documentation can be found here: https://bootstrap-vue.org/docs/components/collapse